### PR TITLE
Remove ngCapsules Dependency

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-14151
Build: https://builds.cask.co/browse/CDAP-UDUT51

### Changes
- Port ngCapsules modules into CDAP
- Changed the module names to conform to CDAP's modules (commons, filters, services)
- Add SockJS client and node-uuid into bower components (before it was being loaded as part of cask-angular-socket-datasource)
- Remove ngCapsules modules from module loader in top level projects (hydrator, tracker, and logviewer)
- Fix linting errors

### CODE REVIEW

File: cdap-ui/Gulpfile.js [hunk 1]
```diff
@@ -238,7 +237,8 @@ gulp.task('js:lib', function() {
       './node_modules/ngreact/ngReact.min.js',
 
       './node_modules/cdap-avsc/dist/cdap-avsc-lib.js',
-      './node_modules/svg4everybody/dist/svg4everybody.min.js'
+      './node_modules/svg4everybody/dist/svg4everybody.min.js',
+      './node_modules/sockjs-client/dist/sockjs.js'
     ].concat([
       './bower_components/cask-angular-*/*/module.js'
```
Comment (ajainarayanan) [inline cdap-ui/Gulpfile.js]: nit: This can be removed I think.

**Review by ajainarayanan — COMMENTED** [review]
@elfenheart One minor comment. Otherwise 👍 LGTM

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

**Comment by elfenheart** [comment]
https://builds.cask.co/browse/CDAP-UDUT51-4

**Pull request merged**